test/functional/test_host_io.py

```diff
@@ -0,0 +1,275 @@
+        os.system('infrasim config add test {}'.format(tmp_conf_file))
+        node = model.CNode(conf)
+        node.init()
+        node.precheck()
+        node.start()
+        time.sleep(3)
+
+    def stop_node(self):
+        global conf
+        node = model.CNode(conf)
+        node.init()
+        node.stop()
+        node.terminate_workspace()
+        conf = {}
+        if os.path.exists(tmp_conf_file):
+            os.unlink(tmp_conf_file)
+        time.sleep(5)
+
+    def port_forward(self):
+        import telnetlib
+        tn = telnetlib.Telnet(host='127.0.0.1', port=2345)
+        tn.read_until('(qemu)')
+        tn.write('hostfwd_add ::2222-:22\n')
+        tn.read_until('(qemu)')
+        tn.close()
+
+    def prepare_ssh(self):
+        ssh = paramiko.SSHClient()
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        paramiko.util.log_to_file('filename.log')
+        helper.try_func(
+            600,
+            paramiko.SSHClient.connect,
+            ssh,
+            '127.0.0.1',
+            port=2222,
+            username='root',
+            password='root',
+            timeout=120)
+        time.sleep(2)
+        return ssh
+
+    def setUp(cls):
+        DOWNLOAD_URL = 'https://github.com/InfraSIM/test/raw/master/image/kcs.img'
+        MD5_KCS_IMG = 'cfdf7d855d2f69c67c6e16cc9b53f0da'
+        try:
+            helper.fetch_image(DOWNLOAD_URL, MD5_KCS_IMG, test_img_file)
+        except InfraSimError as e:
+            print e.value
+            assert False
+
+        cls.start_node()
+        cls.port_forward()
+
+    def tearDown(cls):
+        if conf:
+            cls.stop_node()
+
+    def run_command(
+            cmd='',
+            shell=True,
+            stdout=None,
+            stderr=None,
+            interactive_input=''):
+        child = subprocess.Popen(
+            cmd, shell=shell, stdout=stdout, stderr=stderr)
+        cmd_result = child.communicate(interactive_input)
+        cmd_return_code = child.returncode
+        if cmd_return_code != 0:
+            return (-1, cmd_result[1])
+        return (0, cmd_result[0])
+
+    def test_file_existance_after_node_restart(self):
+        # Write disk
+        node_name = conf['name']
+        ssh = self.prepare_ssh()
+        stdin, stdout, stderr = ssh.exec_command('touch /root/source.bin')
+        while not stdout.channel.exit_status_ready():
+            pass
+        stdin, stdout, stderr = ssh.exec_command(
+            "echo 'Test message is found! :D' >> /root/source.bin")
+        while not stdout.channel.exit_status_ready():
+            pass
+
+        ssh.close()
+        try:
+            ssh = self.prepare_ssh()
+            stdin, stdout, stderr = ssh.exec_command(
+                'dd if=/root/source.bin of=/dev/sdb bs=512 seek=0 count=1 conv=fsync')
+            while not stdout.channel.exit_status_ready():
+                pass
+
+            ssh.close()
+        except Exception as e:
```

Comment:
This try...except... is just get the excepting and raise it, doing nothing. It can be removed.